pkg/operator/etcdmemberscontroller/etcdmemberscontroller.go

```diff
@@ -174,6 +174,20 @@ func (c *EtcdMembersController) reportEtcdMembers() error {
 	return nil
 }
 
+func getMemberMessage(availableMembers, unhealthyMembers, unstartedMembers []string, allMembers []*etcdserverpb.Member) string {
+	messages := []string{}
+	if len(availableMembers) > 0 {
+		messages = append(messages, fmt.Sprintf("%d of %d members are available (%s)", len(availableMembers), len(allMembers), strings.Join(availableMembers, ",")))
```

Comment:
```suggestion
		messages = append(messages, fmt.Sprintf("%d members are available (%s)", len(availableMembers), strings.Join(availableMembers, ",")))
```
We shouldn't need to repeat the count now that we always list available. (ditto for subsequent messages)

Reply:
will update.